test/Microsoft.Azure.WebJobs.Host.EndToEndTests/Scale/ScaleHostEndToEndTests.cs

```diff
@@ -88,58 +83,35 @@ public async Task ScaleManager_GetScaleStatusAsync_ReturnsExpected(bool tbsEnabl
             hostBuilder.ConfigureAppConfiguration((hostBuilderContext, config) =>
             {
                 // Adding host.json here
-                config.AddJsonStream(new MemoryStream(Encoding.ASCII.GetBytes(hostJson)));
+                config.AddJsonStream(new MemoryStream(Encoding.UTF8.GetBytes(hostJson)));
 
                 // Adding app settings
-                config.AddInMemoryCollection(new List<KeyValuePair<string, string>>()
+                config.AddInMemoryCollection(new Dictionary<string, string>
                 {
-                    new KeyValuePair<string, string>("app_setting1", "value1"),
-                    new KeyValuePair<string, string>("app_setting2", "value2")
+                    { "app_setting1", "value1" },
+                    { "app_setting2", "value2"},
+                    { "FeatureManagement:Microsoft.Azure.WebJobs.Host.EndToEndTests", "true" }
                 });
-
-                // Adding hosting config features
-                //config.AddInMemoryCollection(new List<KeyValuePair<string, string>>()
-                //{
-                //    new KeyValuePair<string, string>($"{Constants.HostingConfigSectionName}:Microsoft.Azure.WebJobs.Host.EndToEndTests", "1"),
-                //});
             })
             .ConfigureServices(services =>
             {
-                // Set up IConcurrencyStatusRepository for the test
-                // Adding default AzureCompoentFactory
                 services.AddAzureClientsCore();
-                // Adding IConcurrencyStatusRepository/IAzureBlobStorageProvider
-                services.AddAzureStorageScaleServices(options =>
-                {
-                    options.InternalSasBlobContainer = "sas-blob-container";
-                });
-                // replace IConcurrencyStatusRepository for the test
-                services.AddSingleton<IConcurrencyStatusRepository, TestConcurrencyStatusRepository>();
-                // Adding services for scale host.
-                services.AddSingleton(triggerMetadataProvider);
-                services.AddSingleton<IScaleMetricsRepository, TestScaleMetricsRepository>();
+                services.AddAzureStorageScaleServices();
 
-                // Specific services registrations for scale host
-                services.AddSingleton(triggerMetadataProvider);
+                services.AddSingleton<IConcurrencyStatusRepository, TestConcurrencyStatusRepository>();
```

Comment:
Let's strive to keep the builder usage clean here - I keep cleaning it up, then it gets bloated again.